src/TestExplorer/TestParsers/TestEventStreamReader.ts

```diff
@@ -74,8 +74,7 @@ export class UnixNamedPipeReader implements INamedPipeReader {
                     });
 
                     readable.on("drain", () => pipe.resume());
-
-                    pipe.on("error", () => fs.close(fd));
+                    pipe.on("error", () => pipe.close());
```

Comment:
Drive-by fix, or…?

Reply:
Yeah, the latest VS Code update means when developing the extension if an error is thrown a breakpoint is automatically hit, which would happen here if the `fd` was already closed. Double closing it wasn't really an issue, but now it makes development annoying. `pipe.close()` should do the same thing without throwing if its already closed.